### PR TITLE
v1: expose filesystem customization

### PR DESCRIPTION
**Pull request by chloenayon**

### CODE REVIEW

**Comment by chloenayon** [comment]
currently blocked by https://github.com/osbuild/osbuild-composer/pull/2285

File: internal/v1/api.yaml [hunk 1]
```diff
@@ -532,6 +532,22 @@ components:
...
+      required:
+        - mountpoint
+        - min_size
+      properties:
+        mountpoint:
+          type: string
+          example: '/var'
+        min_size:
+          type: integer
+          example: 1024
```
[the remark this answers is not in the document]
Comment (teg) [reply internal/v1/api.yaml:550]: Agreed, would be nicer. Also the same in composer. Not necessarily a blocker.

[the remark this answers is not in the document]
Comment (kingsleyzissou) [reply internal/v1/api.yaml:550]: Yep can fix it in a follow up.

**Review by kingsleyzissou — APPROVED** [review]

**Comment by Gundersanne** [comment]
It's currently only the installer that's failing here, and that's known to be broken (unrelated to https://github.com/osbuild/osbuild-composer/pull/2340 as well). So I propose we merge this? This is an image issue, not an api issue.

**Comment by teg** [comment]
Agreed.

**Pull request merged**